src/XProfile/VirtoCommerce.XProfile/Schemas/ProfileSchema.cs

```diff
@@ -0,0 +1,228 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GraphQL;
+using GraphQL.Builders;
+using GraphQL.DataLoader;
+using GraphQL.Resolvers;
+using GraphQL.Types;
+using GraphQL.Types.Relay.DataObjects;
+using MediatR;
+using VirtoCommerce.CustomerModule.Core.Model;
+using VirtoCommerce.ExperienceApiModule.Core.Schema;
+using VirtoCommerce.ExperienceApiModule.XProfile.Commands;
+using VirtoCommerce.ExperienceApiModule.XProfile.Queries;
+using VirtoCommerce.ExperienceApiModule.XProfile.Services;
+
+namespace VirtoCommerce.ExperienceApiModule.XProfile.Schemas
+{
+    public class ProfileSchema : ISchemaBuilder
+    {
+        public const string _commandName = "command";
+
+        private readonly IMediator _mediator;
+        private readonly IDataLoaderContextAccessor _dataLoader;
+        private readonly IMemberServiceX _memberService;
+
+        public ProfileSchema(IMediator mediator, IDataLoaderContextAccessor dataLoader, IMemberServiceX memberService)
+        {
+            _mediator = mediator;
+            _dataLoader = dataLoader;
+            _memberService = memberService;
+        }
+
+        public void Build(ISchema schema)
+        {
+            //Queries
+            var organizationFiled = new FieldType
+            {
+                Name = "organization",
+                Arguments = new QueryArguments(new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "organizationId" }),
+                Type = GraphTypeExtenstionHelper.GetActualType<OrganizationType>(),
+                Resolver = new AsyncFieldResolver<object>(async context =>
+                {
+                    var organizationId = context.GetArgument<string>("organizationId");

```

Comment:
user short names for obvious arguments `id` will be enought

Reply:
ok